pycbc/filter/matchedfilter.py

```diff
@@ -298,13 +299,14 @@ def full_matched_filter_and_cluster_fc(self, segnum, template_norm, window):
                                      self.snr_threshold / norm)
         idx, snrv = events.cluster_reduce(idx, snrv, window)
 
-        if len(idx) == 0:
-            return [], [], [], [], []
-
         logging.info("%s points above threshold" % str(len(idx)))
-        return self.snr_mem, norm, self.corr_mem, idx, snrv
 
-    def full_matched_filter_thresh_only(self, segnum, template_norm, window):
+        delta_t = 1.0/(self.delta_f * self.tlen)
+        snr = TimeSeries(self.snr_mem, epoch=epoch, delta_t=delta_t)
```

Comment:
Can you also do copy=False here? And move the delta_t to something in the init of the class.